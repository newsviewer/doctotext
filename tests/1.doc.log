Using DOC parser.
Opening 1.doc as OLE file to parse all embedded objects in supported formats.
No ObjectPool found, embedded OLE objects probably do not exist.
No annotations.
