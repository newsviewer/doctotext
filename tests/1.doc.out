_hyperlink_test_

